.github/workflows/cli.yaml

```diff
@@ -48,11 +45,20 @@ jobs:
           export KUBECONFIG=${HOME}/karmada/karmada-apiserver.config
           GO111MODULE=on go install github.com/onsi/ginkgo/v2/ginkgo
           ginkgo -v --race --trace -p  --focus="[BasicPropagation] propagation testing deployment propagation testing"  ./test/e2e/
-      - uses: chainguard-dev/actions/kind-diag@main
-        # Only upload logs on failure.
-        if: ${{ failure() }}
+      - name: export logs 
+        if: always()
```

Comment:
Is it necessary to collect logs when workflow succeeds or is canceled as well?

Reply:
I hope it can be treated the same as CI, https://github.com/karmada-io/karmada/blob/4898c0fc55a13033fcf21fcc46a41318548c5713/.github/workflows/ci.yml#L158